app/src/main/java/ai/elimu/analytics/db/RoomDb.java

```diff
@@ -228,4 +232,15 @@ public void migrate(@NonNull SupportSQLiteDatabase database) {
             database.execSQL("ALTER TABLE VideoLearningEvent_tmp RENAME TO VideoLearningEvent");
         }
     };
+
+    private static final Migration MIGRATION_13_14 = new Migration(13, 14) {
+        @Override
+        public void migrate(@NonNull SupportSQLiteDatabase database) {
+            Timber.i("migrate (" + database.getVersion() + " --> 14)");
+
+            String sql = "CREATE TABLE IF NOT EXISTS `NumberLearningEvent` (`id` INTEGER PRIMARY KEY AUTOINCREMENT, `androidId` TEXT NOT NULL, `packageName` TEXT NOT NULL, `time` INTEGER NOT NULL, `numberId` INTEGER, `numberValue` INTEGER NOT NULL, `learningEventType` TEXT, `numberSymbol` TEXT NOT NULL, `additionalData` TEXT)";
```

Comment:
@tuancoltech `numberSymbol` can be missing: https://github.com/elimu-ai/webapp/blob/main/src/main/java/ai/elimu/entity/analytics/NumberLearningEvent.java#L24
* http://eng.elimu.ai/content/number/list
* http://tgl.elimu.ai/content/number/list
* http://vie.elimu.ai/content/number/list

```suggestion
            String sql = "CREATE TABLE IF NOT EXISTS `NumberLearningEvent` (`id` INTEGER PRIMARY KEY AUTOINCREMENT, `androidId` TEXT NOT NULL, `packageName` TEXT NOT NULL, `time` INTEGER NOT NULL, `numberId` INTEGER, `numberValue` INTEGER NOT NULL, `learningEventType` TEXT, `numberSymbol` TEXT, `additionalData` TEXT)";
```